content/en/workshop/index.adoc

```diff
@@ -60,14 +61,14 @@ We welcome submissions on, but not limited to, the following topics:
 * Learning from sparse data: Novel methods for learning from small or sparse data in Turkic languages
 * Resources: Datasets, benchmarks, and software libraries for NLP models in Turkic languages
 
-Note: All deadlines are 11:59 pm UTC -12h (“anywhere on Earth”).
+Note: All deadlines are 11:59 pm UTC -12h ("anywhere on Earth").
 
 == "Hack Together" programming event
 
-In addition to the workshop itself, the second day will be devoted to a full-day collaborative programming event "Hack Together".
-Our goal is to demonstrate the SIGTURK NLP library and interested parties can contribute to the integration of new NLP methods and models into the SIGTURK pipeline.
+In addition to the workshop itself, the second day will contain presentations and collaborative events with a particular focus on coding guidelines and practices for behavior-memory safety and test-driven development.
+Our goal is to demonstrate the upbringing of the SIGTURK NLP library, and interested parties can contribute to the integration of new NLP methods and models or their building blocks into the SIGTURK pipeline.
 The SIGTURK infrastructure can be found at https://github.com/sigturk.
-Findings of the event will be combined into a system demonstration paper.
+The findings of the event might be combined into a system demonstration paper.
```

Comment:
A system demo paper would have to be submitted by the workshop deadline, wouldn't it? Could we just say that we will include something about this in the proceedings instead? For example: "The findings of the event may/will be described in the workshop proceedings".

Reply:
I meant subsequently, there's a possibility, but I don't think we should particularly put such a pressure on proceedings, I'd prefer to keep the wording

Reply:
:+1: